src/common/options/crimson.yaml.in

```diff
@@ -102,3 +107,18 @@ options:
   level: dev
   desc: Total size to use for CircularBoundedJournal if created, it is valid only if seastore_main_device_type is RANDOM_BLOCK
   default: 5_G
+- name: seastore_multiple_tiers_stop_evict_ratio
+  type: float
+  level: advanced
+  desc: When the used ratio of main tier is less than this value, then stop evict cold data to the cold tier.
+  default: 0.5
+- name: seastore_multiple_tiers_default_evict_ratio
+  type: float
+  level: advanced
+  desc: start to evict cold data to the cold tier when the used ratio of main tier reach this ratio.
```

Comment:
Begin evicting cold data to the cold tier when the used ratio of the main tier reaches this value.